frontend/src/index.jade

```diff
@@ -211,13 +211,15 @@ html
           .author {{ info.name }} ({{ info.author }}) contribution from {{ info.minDate }} to {{ info.maxDate }}
           .contribution(v-if="isLoaded && files.length!=0")
             .searchbox
-              input.radio-button--search(type="radio", name="filter", value="search", v-on:click="enableSearchBar()")
+              input.radio-button--search(type="radio", name="filter", value="search", :checked="isSearchBar",
+                  v-on:click="indicateSearchBar")
               form.file-picker.mui-form--inline(onsubmit="return false")
-                input#search(type="search", placeholder="Filter by glob", v-on:change="updateFilterSearch", disabled)
-                button#submit-button(type="submit", disabled) Filter
+                input#search(type="search", placeholder="Filter by glob", v-on:change="updateFilterSearch",
+                    v-on:click="indicateSearchBar")
+                button#submit-button(type="submit", v-on:click="indicateSearchBar") Filter
             .fileformat
-              input.radio-button--checkbox(type="radio", name="filter", value="checkboxes", v-on:click="enableCheckBoxes()",
-                  checked)
+              input.radio-button--checkbox(type="radio", name="filter", value="checkboxes", :checked="isCheckBoxes",
```

Comment:
use the full version, instead of the shortform `:checked`, sticking to convention of the code repo